back/engines/commercial/insights/spec/services/insights/category_suggestions_service_spec.rb

```diff
@@ -129,28 +129,35 @@ def create_task_from_zsc_result(zsc_result)
 
     subject(:service) { described_class.new }
 
-    where(:input_body, :result) do
-      'simple body'                                   | 'simple body'
-      '<script> script with bad intentions </script>' | 'script with bad intentions'
-      '<p> line 1<br> line 2<br> line 3 </p>'         | 'line 1 line 2 line 3'
+    where(:input_title, :input_body, :result) do
+      'title'  | 'simple body'                           | 'title. simple body'
```

Comment:
The second time I thought "Is it ruby?" :D 
But still looks nice.

Reply:
Actually, it's totally possible to use `rspec-parameterized` without `TableSyntax` DSL but I opted for the fancy version 👨‍🎤